src/main/java/org/cbioportal/cdd/service/exception/ClinicalAttributeNotFoundException.java

```diff
@@ -23,8 +27,12 @@
     private static final Logger logger = LoggerFactory.getLogger(ClinicalAttributeNotFoundException.class);
 
     public ClinicalAttributeNotFoundException(String columnHeader) {
-        super("Could not find clinical attribute by column header '" + columnHeader + "'.");
-        logger.error("Could not find clinical attribute by column header '" + columnHeader + "'.");
+        super("Could not find clinical attribute(s): " + columnHeader);
+        logger.error("Could not find clinical attribute(s): " + columnHeader);
+    }
+
+    public ClinicalAttributeNotFoundException(List<String> columnHeaders) {
+        this(Joiner.on(", ").join(columnHeaders));
```

Comment:
Java has a built in String.join() which we have been using elsewhere. 
https://docs.oracle.com/javase/8/docs/api/java/lang/String.html